Atomic/syscontainers.py

```diff
@@ -743,6 +743,9 @@ def has_system_container_image(self, img):
         if not repo:
             return False
         try:
+            for i in self.get_system_images(get_all=True):
+                if i['Id'].startswith(img):
+                    return True
```

Comment:
instead of retrieving all the images here, we should check if the ostree branch exists.

If `repo.resolve_rev(imagebranch, False)[0]` fails then you can try to resolve `OSTREE_OCIIMAGE_PREFIX/$img` and see if it exists